src/components/Prompt/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Prompt({ isShowed, children, deleteAction, toggleModal }) {
+  return (
+    isShowed && (
+      <div className="prompt" aria-modal="true" role="dialog" id="dialog1">
+        <br />
```

Comment:
Great that they have implemented accessibility. 👏

Reply:
Thank you so much, Verónica! 😅